operator/flags.go

```diff
@@ -56,8 +56,10 @@ func init() {
 		option.ENITags, "ENI tags in the form of k1=v1 (multiple k/v pairs can be passed by repeating the CLI flag)")
 	option.BindEnv(option.ENITags)
 
-	flags.Int64(option.ENIParallelWorkers, defaults.ENIParallelWorkers, "Maximum number of parallel workers used by ENI allocator")
-	option.BindEnv(option.ENIParallelWorkers)
+	flags.Int64(option.ENIParallelWorkersDeprecated, defaults.ParallelAllocWorkers, "Maximum number of parallel workers used by ENI allocator")
+	flags.MarkDeprecated(option.ENIParallelWorkersDeprecated, fmt.Sprintf("please use --%s", option.ParallelAllocWorkers))
+	flags.Int64(option.ParallelAllocWorkers, defaults.ParallelAllocWorkers, "Maximum number of parallel IPAM workers")
```

Comment:
The `option.BindEnv(option.ENIParallelWorkers)` was accidentally removed. Also, please add a new line between lines 60 and 61 for better readability.

Reply:
https://github.com/cilium/cilium/pull/10089#discussion_r385744697